src/environments/environment.ts

```diff
@@ -21,14 +21,17 @@
 export const environment = {
   production: false,
   firebase: {
-    apiKey: '{YOUR_API_KEY}',
-    authDomain: 'yourproject.firebaseapp.com',
-    databaseURL: 'https://yourproject.firebaseio.com',
-    projectId: '{YOUR_PROJECT_ID}',
-    storageBucket: 'undefined',
-    messagingSenderId: '{YOUR_MESSAGING_SENDER_ID}',
-    appId: '{YOUR_APP_ID}',
-    measurementId: '{YOUR_MEASUREMENT_ID}',
+    apiKey: "AIzaSyCliUfJo15BWUEHpWPr1NTgtyMxnlgPOnk",
+
+    authDomain: "cfa-p-jigsaw.firebaseapp.com",
+
+    projectId: "cfa-p-jigsaw",
+
+    storageBucket: "cfa-p-jigsaw.appspot.com",
+
+    messagingSenderId: "201317601635",
+
+    appId: "1:201317601635:web:95a28f99599f5561570356"
```

Comment:
Can you please remove this asap and add a story to change all these since they're now completely exposed?

---

Add them to gitignore if it's problem.